lib/factory_girl/linter_base.rb

```diff
@@ -0,0 +1,49 @@
+module FactoryGirl
+  class LinterBase
```

Comment:
Sure, if this is what you want. I was trying to preserve the existing code and not make this relative small feature have too much complexity to it. But yeah, a strategy would be better and my preference as well.